modules/gridmap/grid_map.cpp

```diff
@@ -1121,6 +1160,8 @@ void GridMap::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "collision_priority"), "set_collision_priority", "get_collision_priority");
 	ADD_GROUP("Navigation", "");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "bake_navigation"), "set_bake_navigation", "is_baking_navigation");
+	ADD_GROUP("Rendering", "rendering_");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "rendering_layer", PROPERTY_HINT_LAYERS_3D_RENDER), "set_rendering_layer_mask", "get_rendering_layer_mask");
```

Comment:
```suggestion
	ADD_PROPERTY(PropertyInfo(Variant::INT, "rendering_layers", PROPERTY_HINT_LAYERS_3D_RENDER), "set_rendering_layer_mask", "get_rendering_layer_mask");
```
To match other use, it's `layers` in `VisualInstance3D`, and it's not a singular layer